dot-studio/svc_coord

```diff
@@ -33,23 +33,23 @@ document "wait_for_success" <<DOC
   TIMEOUT=60 wait_for_success curl localhost:8080/status
 DOC
 function wait_for_success() {
-  local SECONDS_WAITING=${TIMEOUT:-60};
-  local COUNTER=0;
+  local SECONDS_WAITING=${TIMEOUT:-60}
+  local COUNTER=0
 
   while : ; do
-    if eval "$@" >/dev/null 2>/dev/null
-    then
+    if eval "$@" >/dev/null 2>/dev/null; then
       break
     else
       sleep 1
     fi
-    echo " => Waiting for '$*' to succeed. ($COUNTER of $SECONDS_WAITING)";
+    echo " => Waiting for '$*' to succeed. ($COUNTER of $SECONDS_WAITING)"
 
-    if [ $COUNTER -ge $SECONDS_WAITING ]; then
-      echo " => ERROR: Command '$@' never succeeded in $SECONDS_WAITING seconds.";
+    if [[ $COUNTER -ge "$SECONDS_WAITING" ]]; then
+      # shellcheck disable=2145
+      echo " => ERROR: Command '$@' never succeeded in $SECONDS_WAITING seconds."
```

Comment:
Change `'$@'` to `'$*'` and I think the `shellcheck disable` can be removed. This is inconsistent with line 47 anyway.